src/screens/scanning/ReviewScreen.tsx

```diff
@@ -99,7 +101,11 @@ export default function ReviewScreen({
   };
 
   const onDeleteHelper = () => {
+    const { size } = voucherMap;
     deleteVoucher(dispatch, focusedSerialNumber);
+    if (size <= 1) {
```

Comment:
I wonder if it's really best to lump this with onDelete functionality? It was pretty jarring to me when the dialog suddenly popped up after I deleted the voucher and forced me to return to the start. Personally I would've expected to see that kind of error dialog if I tried to submit the empty invoice, but not the moment the invoice becomes empty.

Reply:
Yeah, I see your point here. @ronniebeggs let's display the error dialog on clicking the Submit button instead.